src/fromager/commands/build.py

```diff
@@ -101,13 +101,20 @@ def build_sequence(
             source_download_url = entry["source_url"]
             req = Requirement(f"{dist_name}=={resolved_version}")
 
+            log_filename = str(resolved_version) + "-" + dist_name + ".log"
+            wheel_log = wkctx.work_dir / log_filename
+
+            file_handler = logging.FileHandler(str(wheel_log), mode="w")
+            logger.addHandler(file_handler)
```

Comment:
Doing the logging here is going to cause us to produce a log with the skipping messages on line 112. We don't want to upload those and overwrite other log files that have useful information. Can we do the logging down in the `_build()` function, where we know we're going to try to build the wheel?